api/src/main/java/marquez/db/EventDao.java

```diff
@@ -0,0 +1,37 @@
+package marquez.db;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import marquez.db.mappers.RawLineageEventMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+@RegisterRowMapper(RawLineageEventMapper.class)
+public interface EventDao extends BaseDao {
```

Comment:
Can we query for events using [`OpenLineageDao`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/db/OpenLineageDao.java)?

```
OpenLineageDao.findAllEvents()
OpenLineageDao.findAllEventsByNamespace()
```

`OpenLineageDao` does need some refactoring, but given that we are interacting with lineage events, it would make sense to reuse the class vs defining a separate DAO.

Reply:
Done.